test_examples.sh

```diff
@@ -2,153 +2,10 @@
-
-# gym/ppo batch (specify non-mujoco env to test without mujoco)
-python examples/gym/train_ppo_batch_gym.py --steps 100 --update-interval 50 --batchsize 16 --epochs 2 --outdir $outdir/gym/ppo_batch --env Pendulum-v0 --gpu $gpu
-model=$(find $outdir/gym/ppo_batch -name "*_finish")
-python examples/gym/train_ppo_batch_gym.py --demo --load $model --eval-n-runs 1 --env Pendulum-v0 --outdir $outdir/temp --gpu $gpu
-
-# gym/trpo (specify non-mujoco env to test without mujoco)
-python examples/gym/train_trpo_gym.py --steps 100 --trpo-update-interval 50 --outdir $outdir/gym/trpo --env Pendulum-v0 --gpu $gpu
-model=$(find $outdir/gym/trpo -name "*_finish")
-python examples/gym/train_trpo_gym.py --demo --load $model --eval-n-runs 1 --env Pendulum-v0 --outdir $outdir/temp --gpu $gpu
-
-# gym/categorical_dqn
-python examples/gym/train_categorical_dqn_gym.py --steps 100 --replay-start-size 50 --outdir $outdir/gym/categorical_dqn --gpu $gpu
-model=$(find $outdir/gym/categorical_dqn -name "*_finish")
-python examples/gym/train_categorical_dqn_gym.py --demo --load $model --eval-n-runs 1 --outdir $outdir/temp --gpu $gpu
-
-# gym/iqn
-python examples/gym/train_iqn_gym.py --steps 100 --replay-start-size 50 --outdir $outdir/gym/iqn --gpu $gpu
-model=$(find $outdir/gym/iqn -name "*_finish")
-python examples/gym/train_iqn_gym.py --demo --load $model --eval-n-runs 1 --outdir $outdir/temp --gpu $gpu
-
-# grasping/dqn
-python examples/grasping/train_dqn_batch_grasping.py --gpu $gpu --steps 100 --outdir $outdir/grasping/dqn
-model=$(find $outdir/grasping/dqn -name "*_finish")
-python examples/grasping/train_dqn_batch_grasping.py --demo --load $model --eval-n-runs 1 --outdir $outdir/temp --gpu $gpu
-
-# mujoco/ppo (specify non-mujoco env to test without mujoco)
-python examples/mujoco/ppo/train_ppo.py --steps 10 --update-interval 5 --batch-size 5 --epochs 2 --outdir $outdir/mujoco/ppo --env Pendulum-v0 --gpu $gpu
-model=$(find $outdir/mujoco/ppo -name "*_finish")
-python examples/mujoco/ppo/train_ppo.py --demo --load $model --eval-n-runs 1 --env Pendulum-v0 --outdir $outdir/temp --gpu $gpu
-
-# mujoco/ddpg (specify non-mujoco env to test without mujoco)
-python examples/mujoco/ddpg/train_ddpg.py --env Pendulum-v0 --gpu $gpu --steps 10 --replay-start-size 5 --batch-size 5 --outdir $outdir/mujoco/ddpg
-model=$(find $outdir/mujoco/ddpg -name "*_finish")
-python examples/mujoco/ddpg/train_ddpg.py --env Pendulum-v0 --demo --load $model --eval-n-runs 1 --outdir $outdir/temp --gpu $gpu
-
-# mujoco/td3 (specify non-mujoco env to test without mujoco)
-python examples/mujoco/td3/train_td3.py --env Pendulum-v0 --gpu $gpu --steps 10 --replay-start-size 5 --batch-size 5 --outdir $outdir/mujoco/td3
-model=$(find $outdir/mujoco/td3 -name "*_finish")
-python examples/mujoco/td3/train_td3.py --env Pendulum-v0 --demo --load $model --eval-n-runs 1 --outdir $outdir/temp --gpu $gpu
+for SCRIPT in $(find examples_tests/ -type f -name '*.sh')
```

Comment:
```suggestion
for SCRIPT in $(find examples_tests/ -type f -name 'test_*.sh')
```
if non-test scripts will be added under the directory.